api/app/tests/apps/services/images.py

```diff
@@ -0,0 +1,48 @@
+from io import BytesIO
```

Comment:
This test file should be in `api/app/tests/apps/services/domain/images.py` instead of `api/app/tests/apps/services/images.py`. Then it will match the place of the actual code

Reply:
moved as per suggestion